codespell_lib/data/dictionary.txt

```diff
@@ -7260,26 +7688,40 @@ avarageing->averaging
 avare->aware
 avarege->average
 avary->every, aviary,
+avatara->avatar
+avataras->avatars
+avatards->avatars
+avatares->avatars
 avation->aviation
 avcoid->avoid
 avcoids->avoids
 avdisories->advisories
 avdisoriy->advisory, advisories,
 avdisoriyes->advisories
 avdisory->advisory
-avengence->a vengeance
+avengence->a vengeance, vengeance,
+avent->advent, event,
+averadge->average
+averadges->averages
+averageadi->averaged
 averageed->averaged
+averageifs->averages
```

Comment:
Can't find this typo in the wild. It doesn't make sense to include it. Besides, that's an Excel function:
https://support.microsoft.com/en-us/office/averageifs-function-48910c45-1fc0-4389-a028-f7c5c3001690